frontend/src/views/LevelView.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+  <div class="PageWrapper">
+    <div class="Container">
+      <img src="@/assets/images/logo-and-lettering.svg" alt="Saci Logo" />
+      <h1>{{ newLevelTitle }}</h1>
+      <h2>Basic Information</h2>
+      <div class="NameContainer">
+        <h5>Name</h5>
+        <ElInput placeholder="Level Name" v-model="newLevel.name" />
+      </div>
+      <div class="FlexContainer">
+        <div class="MaxAndMinInputs">
+          <h5>Coeficient Range</h5>
+          <span>
+            <ElInput placeholder="Min Range" v-model="newLevel.minCoeficient"></ElInput>
+            <h4>to</h4>
+            <ElInput placeholder="Max Range" v-model="newLevel.maxCoeficient"></ElInput>
+          </span>
+        </div>
+        <div class="LinkSide">
+          <h5>Link to Knowledge Path</h5>
+          <ElInput placeholder="Link" v-model="newLevel.link"></ElInput>
+        </div>
+      </div>
+      <div class="SaveButton">
+        <ElButton type="primary" @click="createLevel()"> Save </ElButton>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import type { Role } from '@/domain/Role'
+import type { Level } from '@/domain/Level'
+import { useRolesStore } from '@/stores/roles'
+import { ref, onMounted, computed } from 'vue'
+import { ElInput, ElButton } from 'element-plus'
+import { ErrorMessage, SuccessMessage } from '@/services/messages'
+import { useRouter } from 'vue-router'
+import { HttpServerError } from '@/services/http'
+
+const roleStore = useRolesStore()
+const roleId = ref<string>('1')
```

Comment:
You don't need this variable, just to mock. Pass the value directly in the function to get the role